File: Curso Logica de Programação Neri Neitzke/Algoritmos em Java/tabuada_while.java
// Prof Neri Aldoir Neitzke   ULBRA
// www.INFORMATICON.com.br - videoaulasneri@gmail.com
/*Problema: Ler uma variável de numero inteiro e mostrar a tabuada desse número.*/
import javax.swing.JOptionPane;
class tabuada_while
{
   public static void main(String args[])
   {
        int i, numero;
        String mensagem = "";
        numero = Integer.parseInt(JOptionPane.showInputDialog("Digite um numero.:"));
        i = 1;

        while(i <= 9)      //i++ é igual a i=i+1;
        {
            mensagem = mensagem + "\n"+numero + " x "+i+" = "+(numero * i);
            i++;     //igual ao i=i+1;
        }
        System.out.println(mensagem);
        JOptionPane.showMessageDialog(null, mensagem);

  }
}

File: Curso Logica de Programação Neri Neitzke/Algoritmos em Pascal/pascal/tabuada_while.pas
// Prof Neri Aldoir Neitzke         - ULBRA
// www.INFORMATICON.com.br - videoaulasneri@gmail.com
{Problema: Ler uma variável de numero inteiro e mostrar a tabuada desse número. }
program tabuada_while;
var numero,i: integer;
begin
    write('Digite o numero : ');
    readln(numero);
    i := 1;
    while(i <= 9) do
    begin
        writeln(numero  ,' x ',i,' = ',(numero * i));
        i:= i +1;
    end;
    writeln('..................Final do Sistema..................... ');
    readln();
end.
